external/fv3fit/tests/reservoir/test_domain.py

```diff
@@ -1,6 +1,12 @@
 import numpy as np
+import os
 import pytest
-from fv3fit.reservoir.domain import slice_along_axis
```

Comment:
Nothing is new here, after deleting `test_1d_domain` I consolidated the tests in `test_domain` and `test_cubedsphere_domain` into a single test module.